qemu/tests/cfg/multi_disk_random_hotplug.cfg

```diff
@@ -1,19 +1,37 @@
 - multi_disk_random_hotplug: install setup image_copy unattended_install.cdrom
     type = multi_disk_random_hotplug
+    start_vm = no
+    not_preprocess = yes
     force_create_image = yes
     force_create_image_image1 = no
     remove_image = yes
     remove_image_image1 = no
     stg_image_name = "images/stg%s"
     stg_image_num = 20
+    stg_image_size = 500M
```

Comment:
Could it use less disk,like 128M, you know some host have poor disk space.

Reply:
Okay， will update it